_drafts/autolab-1-7-0.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Autolab 1.7.0"
+author: 'Jake Zimmerman'
+# Some sample tags; you'll probably want to use different ones.
+tags: ['release']
+comments: true   # toggle this to disable comments
+# Be sure to read _drafts/docs-markdown-style-guide/ :D
+---
+
+Hey everyone! We've been hard at work this summer getting Autolab primed for
+the upcoming fall semester. Here's a quick rundown of what's new or changed in
+this release.
+
+
+# Annotate student-submitted PDFs
```

Comment:
I tried but

- I couldn't find a class currently using it
- I think staging is using the older version of PDF commenting
- Everything on production is broken right now /cc @icanberk